Captain-AJ.md

```diff
@@ -0,0 +1,54 @@
+# name: "@Captain-AJ"
+# project: "An autonomous butler robot"
+
+# Project Name
+A.L.F.R.E.D. Butlerbot
+
+## Summary
+
+I plan to build an Arduino controlled autonomous robot butler named A.L.F.R.E.D. (Advanced, Little, Fit, Robot, Education, Device) that can be summoned at the press of a 
+button and deliver items. I am excited because this practical project has been on my list for sometime now. 
+After a long part search, my efforts to build this contraption summed in the listed Devastator robot kit. The set includes treads, metal motors, an aluminum tank chassis, 
+and a battery compartment.
+All coincides with my desire to apply computer and mechanical engineering concepts to develop a computing module in physical excellence.
+
+## Plan
+
+1. Order kit and required parts.
+
+2. Read a sizable amount of Arduino Cookbook to become well endowed with appropriate knowledge.
+
+3. Build the kit according to the instructions: 
+a. Assemble the tracks with gear components. 
+b. Assemble the robot's chassis and attach the Romeo BLE.
+c. Attach all sensors to the body, e.g., Ultrasonic sensor, speaker, etc.
+d. Pan/tilt systems for ultrasonic sensor and camera
+e. Wire all components 
+f. Attach the food storage unit.
+
+4. Programing (Python, C language):
+a. PixyCamera for color and facial recognition.
+b. Ultrasonic sensor for object detection and avoidance.
+c. Speaker for communication.
+d. OLED display for facial expressions and personality.
+e. Gyroscope for absolute direction capabilities, e.g. field-oriented drive.
+f. Unify sensor control by programming manual and autonomous control functions.
+
+    The robot is now complete!
+
+
```

Comment:
```suggestion
| Product         | Supplier/Link                                         | Cost    |
| ------------- | ------------------------------------ | ------- |
```

Reply:
I plan to read the Arduino Cookbook, ask engineering advisers questions relating to cases that are specific for my build, and research online.

Furthermore, I will also use the listed tutorials from [dfrobot.com](http://dfrobot.com/) to assist me:

Explaining Computers:

[[Raspberry Pi Devastator Robot #1](https://youtu.be/DyQxvfwQbTg)](https://youtu.be/DyQxvfwQbTg)

[[Raspberry Pi Devastator Robot #2](https://youtu.be/j6mglfhWZrQ)](https://youtu.be/j6mglfhWZrQ)

[[Raspberry Pi Devastator Robot #3: Camera](https://youtu.be/pK0XvjiP2qk)](https://youtu.be/pK0XvjiP2qk)

[educ8s.tv](http://educ8s.tv/):

[[Arduino Tank Robot Project using the Devastator metal chassis!](https://youtu.be/RTFSzXnlx4E)](https://youtu.be/RTFSzXnlx4E)

Reply:
great! could you please commit this into your file? thank you sm :)

Reply:
I have updated the file.